apps/ensadmin/src/components/indexing-status/view-models.ts

```diff
@@ -116,12 +136,14 @@ export function networkIndexingStatusViewModel(
   });
 
   return {
+    id: chainId,
     name: chainName,
     latestSafeBlock: blockViewModel(latestSafeBlock),
     firstBlockToIndex: blockViewModel(firstBlockToIndex),
     lastIndexedBlock: lastIndexedBlock ? blockViewModel(lastIndexedBlock) : null,
     lastSyncedBlock: lastSyncedBlock ? blockViewModel(lastSyncedBlock) : null,
     phases,
+    ...(chainsBlockExplorer && { blockExplorer: chainsBlockExplorer }),
```

Comment:
Let's make this one not optional and provide consistent UX for all chains 👍 
```suggestion
    blockExplorer: chainsBlockExplorer,
```